spec/hashie/extensions/indifferent_access_with_rails_hwia_spec.rb

```diff
@@ -35,6 +36,10 @@ class CoercableHash < Hash
     include Hashie::Extensions::MergeInitializer
   end
 
+  class MashWithIndifferentAccess < Hashie::Mash
```

Comment:
I"m not entirely convinced, since this issue only arises when you mix both ways of indifference.
Without Rails' hwia this wouldn't fail.